tsl/test/sql/include/cagg_on_cagg_validations.sql

```diff
@@ -0,0 +1,36 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+--
+-- CAGG on hypertable (1st level)
+--
+CREATE MATERIALIZED VIEW :CAGG_NAME_1ST_LEVEL
+WITH (timescaledb.continuous, timescaledb.materialized_only=true) AS
+SELECT
+  time_bucket(:BUCKET_WIDTH_1ST, "time") AS bucket,
+  SUM(temperature) AS temperature
+FROM conditions
+GROUP BY 1
+WITH NO DATA;
+
+--
+-- CAGG on CAGG (2th level)
+--
+\set ON_ERROR_STOP 0
+\echo :WARNING_MESSAGE
```

Comment:
Not sure what you need this for. It will be clear when you run the test and the error message will be in the output file, hence the expected error message after you have committed the PR. I think you can remove this line and the variable settings.

Reply:
Added this message just for make our life easier when reading/underastangint the output test file. IMHO leaving it as is will not hurt us too much.